src/libs/antares/writer/zip_writer.cpp

```diff
@@ -82,12 +96,14 @@ ZipWriter::ZipWriter(std::shared_ptr<Yuni::Job::QueueService> qs,
  pQueueService(qs),
  pState(ZipState::can_receive_data),
  pArchivePath(std::string(archivePath) + ".zip"),
- pDurationCollector(duration_collector)
+ pDurationCollector(duration_collector),
+ pendingTasks_()
 {
     mz_zip_writer_create(&pZipHandle);
     if (int32_t ret = mz_zip_writer_open_file(pZipHandle, pArchivePath.c_str(), 0, 0); ret != MZ_OK)
     {
         logs.error() << "Error opening zip file " << pArchivePath << " (" << ret << ")";
+        logErrorAndThrow("Error opening zip file " + pArchivePath + " (" + std::to_string(ret) + ")");
```

Comment:
Please remove the previous line (redundant)

Reply:
Removed